src/Core/Types/TypeBase.cs

```diff
@@ -64,7 +64,7 @@ protected virtual string GetNameInProperFormat(string name, int id)
         }
 
         /// <summary>
-        /// check whether the contant should be part of List
+        /// check whether the content should be part of List
```

Comment:
Is this content or constant?

Reply:
Content seems to make more sense. The id is just a key into a collection